HW_1/task2.py

```diff
@@ -0,0 +1,18 @@
+#.Напишите код, который запрашивает число и сообщает является ли оно простым или составным. 
+#Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+#Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+#
+LIMIT_MAX = 100000
+LIMIT_MIN = 0
+n = int(input('Введите число для проверки является ли оно простым : '))
+while  not(LIMIT_MIN < n < LIMIT_MAX):
+    print ('Вы ввели число не из допустимого диапазона от 0 до 100 000')
+    n = int(input())
+count = 0
+for i in range(1, n+1):
```

Comment:
Слишком много итераций. Вспомните теорему из теории чисел, которая гласит, что для любого составного числа есть, по крайней мере, один делитель, не превосходящий корень из этого числа.
```suggestion
for i in range(1, n + 1):
```